lib/pbio/src/protocol/pybricks.c

```diff
@@ -85,6 +85,9 @@ const uint8_t pbio_pybricks_hub_capabilities_char_uuid[] = {
 /** Bluetooth Device Information Service UUID. */
 const uint16_t pbio_gatt_device_info_service_uuid = 0x180A;
 
+/** Bluetooth Device Name UUID. */
+const uint16_t pbio_gatt_device_name_uuid = 0x2A00;
```

Comment:
```suggestion
/** Bluetooth Device Name characteristic UUID. */
const uint16_t pbio_gatt_device_name_char_uuid = 0x2A00;
```

To be consistent with existing UUIDs.